### PR TITLE
remove txn_id label for all metrics

### PR DESCRIPTION
**Pull request by SpontaneousOverthrow**
## The Problem

Adding label `TransactionID` leads to infinity increasing number of unique metrics.

## The Solution

Remove `TransactionID` for all metrics:

- totalTransferredVolume
- receivedAuction
- attemptedAuction
- successfulAuction
- attemptedTransfer
- completedTransfer
- feesCollected
- gasConsumed
- relayerFeesPaid
- senderPrepared
- receiverPrepared
- senderCancelled
- receiverCancelled
- senderFulfilled
- receiverFulfilled
- senderExpired
- receiverExpired
- receiverFailedPrepare
- senderFailedFulfill
- senderFailedCancel
- receiverFailedCancel
- liquiditySupplied

## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [ ] Update documentation if needed.
- [ ] Update CHANGELOG.md.

### CODE REVIEW

File: packages/router/src/lib/helpers/metrics.ts [hunk 1]
```diff
@@ -322,7 +322,6 @@ export const incrementFees = async (
 
   const { requestContext, methodContext } = createLoggingContext(incrementFees.name, _requestContext);
   logger.debug("Method start", requestContext, methodContext, {
-    transactionId,
```
Comment (SpontaneousOverthrow) [inline packages/router/src/lib/helpers/metrics.ts:326]: dirty. Need to save transactionId in debug logs

File: packages/router/src/lib/helpers/metrics.ts [hunk 2]
```diff
@@ -334,7 +333,6 @@ export const incrementFees = async (
 
   if (amount.isNegative()) {
     logger.warn("Got negative fees, doing nothing", requestContext, methodContext, {
-      transactionId,
```
Comment (SpontaneousOverthrow) [inline packages/router/src/lib/helpers/metrics.ts:338]: dirty. Need to save transactionId in warn logs

File: packages/router/src/lib/helpers/metrics.ts [hunk 3]
```diff
@@ -349,7 +347,6 @@ export const incrementFees = async (
   const fees = await convertToUsd(assetId, chainId, amount.toString(), requestContext);
 
   logger.debug("Got fees in usd", requestContext, methodContext, {
-    transactionId,
```
Comment (SpontaneousOverthrow) [inline packages/router/src/lib/helpers/metrics.ts:353]: dirty. Need to save transactionId in debug logs

**Comment by SpontaneousOverthrow** [comment]
Need to fix tests too

**Comment by SpontaneousOverthrow** [comment]
Seems everything is ok, ready for testing in testnet.

**Review by rhlsthrm — APPROVED** [review]

**Comment by rhlsthrm** [comment]
@SpontaneousOverthrow this looks good. Can you run this on your own router using `ROUTER_VERSION=db3e0d6`?

**Pull request merged**